addons/l10n_ch/report/swissqr_report.xml

```diff
@@ -20,50 +20,80 @@
         </template>
 
         <template id="l10n_ch_swissqr_template">
+            <t t-set="o" t-value="o.with_context(lang=lang)"/>
             <t t-call="web.external_layout">
                 <!-- add class to body tag -->
                 <script>document.body.className += " l10n_ch_qr";</script>
 
-                <t t-set="formated_amount" t-value="('%.2f' % (o.amount_residual)).split(' ')[0].replace(',',' ')"/>
+                <t t-set="formated_amount" t-value="'{:,.2f}'.format(o.amount_residual).replace(',','\xa0')"/>
 
                 <div class="swissqr_title">
-                    <h1>QR-bill for invoice <t t-esc="invoice.name"/></h1>
+                    <h1>QR-bill for invoice <t t-esc="o.name"/></h1>
                 </div>
 
                 <div class="swissqr_content">
 
+<<<<<<< HEAD
```

Comment:
:eyes:

Reply:
IMO all can be replaced by the new version. I don't see an element that was added that caused the conflict.